### DISCUSSION TITLE
Best tool to enforce US English spelling?

### DISCUSSION THREAD

**Discussion by hamishwillee** [mdn/mdn-community#534] [Content]
We fixed most cases of non-US-English spelling in https://github.com/mdn/content/issues/373 but while contributors continue to insist on being from "the rest of the world" these will keep on being injected. 

What is the best tool to automate fixing this? If anyone has ideas I'm happy to then post on yari.

FYI @schalkneethling

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
